python/lsst/pipe/tasks/ingestDefects.py

```diff
@@ -0,0 +1,51 @@
+from .ingestCalibs import IngestCalibsTask
+from lsst.obs.base.read_defects import read_all_defects
+from lsst.pipe.base import InputOnlyArgumentParser
+
+import tempfile
+import shutil
+import os
+
+
+class IngestDefectsArgumentParser(InputOnlyArgumentParser):
+    """Argument parser to support ingesting calibration images into the repository"""
+
+    def __init__(self, *args, **kwargs):
+        InputOnlyArgumentParser.__init__(self, *args, **kwargs)
+        self.add_argument("-n", "--dry-run", dest="dryrun", action="store_true",
+                          default=False, help="Don't perform any action?")
+        self.add_argument("--create", action="store_true", help="Create new registry?")
+        self.add_argument("--ignore-ingested", dest="ignoreIngested", action="store_true",
+                          help="Don't register files that have already been registered")
+        self.add_argument("root", help="Root directory to scan for defects.")
+
+
+class IngestDefectsTask(IngestCalibsTask):
+    """Task that generates registry for calibration images"""
```

Comment:
This doc string is wrong because this only handles defects. Consider copying the "run" doc string, since "Task that..." is not very helpful (it is clearly a task) and doc strings are supposed to be in active voice.

At one time task doc strings were supposed to be extensive, with sections for examples, debugging (which debug flags are supported)...is that still the case?